src/utils/iqeEnablement.ts

```diff
@@ -207,6 +210,7 @@ export function init(chromeStore: ReturnType<typeof createStore>, authRef: React
 
 const qe = {
   init,
+  isExcluded,
```

Comment:
@charlesmulder not blocking, but the name `isExcluded` is not very descriptive on _what_ exactly we are excluding and why (outside the context of this file). We may eventually want to update the name of this function to be a bit more clear to others if we expose it. 

Past that, @catastrophe-brandon could you forsee any reason this would cause issues with IQE? I don't think it would, but I just wanted to check.

Reply:
@florkbr That's good thinking. Any name suggestions?

Reply:
@florkbr How about `isAuthHeaderDisabled`? It's a quick addition to this PR.